ts/api/backend.ts

```diff
@@ -201,15 +201,15 @@ export function BackendClient(
 
   const verificaRptT: GetPaymentInfoT = {
     method: "get",
-    url: ({ rptId }) => `/api/v1/payment-requests/${rptId}`,
+    url: ({ rptId, test }) => `/api/v1/payment-requests/${rptId}?${test}`,
```

Comment:
this will append the value to the query, I guess you should add the parameter name too

 ```suggestion
    url: ({ rptId, test }) => `/api/v1/payment-requests/${rptId}?test=${test}`,
```